### ISSUE TITLE
Nanopolish Killed: 9, empty VCF

### ISSUE THREAD

**Issue by Chjulian** [jts/nanopolish#255]
Hi everyone,

I am having an issue running nanopolish 0.8.3

I run Albacore version 2.0.2 and clean the fastq with porechop. 
I use "bwa -× ont2d" and SAM to align/sort/index the cleaned fastq to the reference
I index the cleaned fastq with nanopolish and then run nanopolish eventalign. 

Everything seems ok until this point, I get for example:

> [post-run summary] total reads: 40337 unparseable: 0 qc fail: 7 could not calibrate: 10 no alignment: 489

However, when I run nanopolish variants,  the program runs for a while and then I got "Killed: 9" printed to the screen and an empty vcf.

This is a summary of the instructions
```
nanopolish index --directory /Volumes/fast5 barcode01.fastq 

nanopolish eventalign --threads 8 --sam --reads barcode01.fastq --bam barcode01.sorted.bam --genome reference.fasta | samtools sort > barcode01.eventalign.sorted.bam && samtools index barcode01.eventalign.sorted.bam 

nanopolish variants --threads 8 --reads barcode01.fastq --bam barcode01.sorted.bam --genome reference.fasta --event-bam barcode01.eventalign.sorted.bam --outfile barcode01.eventalign.vcf --ploidy 1
```
Before Albacore 2 and an older version of nanopolish this worked fine (without the nanopolish indexing of course).

Any ideas of what am I missing?

Cheers,
Julian

**Comment by bc2zb** [comment]
I'm fairly certain that whenever you call nanopolish variants, among the first things it does it make an empty VCF. I've run variants multiple times, and the empty VCF always shows up regardless of the results of the variants call. 

You don't need to use eventalign anymore https://github.com/jts/nanopolish/issues/251.

I also cannot get nanopolish variants to run successfully on porechopped fastq files, but have never seen the error you describe.

**Comment by jts** [comment]
Hi Julian,

It looks like you are running on the entire reference genome at once. In this case nanopolish will try to load all 40,000 reads into memory and probably run out of memory. Try calling variants in a smaller window.

@bc2zb with nanopolish 0.8 and above should work with porechop reads

**Comment by Chjulian** [comment]
@bc2zb thanks for the info regarding nanopolish variants

@jts you are right, it is a memory issue but does it means that previous versions of nanopolish loaded reads differently? (previously, I ran the same barcoded samples with a previous version and it was fine)

A smaller window seems not to be an option in my case. To put the data in context, I barcoded the viral gene of 12 samples and even after running MinION for a couple of hours I got extremely high coverage for each of them. So, I wonder if it makes sense to somehow subset the reads from the whole run?

![screen shot 2017-11-06 at 10 18 44](https://user-images.githubusercontent.com/7526558/32434089-63cd4a5a-c2dd-11e7-9e37-86b6940f8c5b.png)
